spec/rails_helper.rb

```diff
@@ -740,6 +772,14 @@ def using_session_with_localhost_resolution(name, &block)
     Scheduler::Defer.do_all_work
   end
 
+  $playwright_logger = nil
```

Comment:
I don’t think we need to do that as `$playwright_logger` will return `nil` by default.